recipe/meta.yaml

```diff
@@ -29,6 +29,9 @@ requirements:
     - traitlets
 
 test:
+  requires:
+    # This is here until the migration to 1.1 is finished.
+    - openssl 1.0
```

Comment:
@ocefpaf, all PyQt packages in conda-forge are broken until PyQt 5.9 is uploaded. So I think we should move Qt 5.9 to beta in the meantime.

Reply:
I mean, to the beta channel.

Reply:
Sorry, I think the problem is not with Qt. It's just that the entire stack moved to OpenSSL 1.1 except Qt.

Reply:
Don't do this or it will confuse, let's wait for pyqt with the right qt and openssl packages, then this one will be OK.